lib/generate-plugin/index.js

```diff
@@ -1,13 +1,13 @@
-var yeoman = require("yeoman-environment");
-var PluginGenerator = require("../generators/plugin-generator").PluginGenerator;
+const yeoman = require("yeoman-environment");
+const PluginGenerator = require("../generators/plugin-generator").PluginGenerator;
```

Comment:
Can you change to `const { PluginGenerator }`

Reply:
Can we use ES6 destructing now? Cool!